app/forms/bulk_submission_form.rb

```diff
@@ -18,6 +29,9 @@ class BulkSubmissionForm
            :file_too_big,
            :file_content_type
 
+  validate :file_content, if: proc {
```

Comment:
💯

Reply:
question: are we confident that the order of validations means that any errors from other validations are added by the time this fires

Reply:
Yes, I have tested that the file_content method is not called if the initial validations fail (e.g. for a png)